app/decorators/book_decorator.rb

```diff
@@ -0,0 +1,25 @@
+class BookDecorator < Draper::Decorator
```

Comment:
why not Rectify::Presenter ?

Reply:
Used in Active Admin Book config, e.g. [here](https://github.com/evtik/book_store/blob/c7855bb5c2afeaee80af83169a8b4f261f74bf83/app/admin/book.rb#L13).